docs/UserGuide.md

```diff
@@ -53,34 +53,58 @@ command to run the application.<br>
 
 ### Adding a client: `add`
 
-Adds a client to the Realodex. Note that Name, Address, Income, Phone Number, Family Size, Email and Tag are compulsory. You may input the parameters in any order (e.g. if 
-the command specifies `n/NAME a/ADDRESS`, `a/ADDRESS n/NAME` is also acceptable). A client can be a buyer, seller or both. This should be reflected by tagging `t/buyer` and/or `t/seller`.
+Adds a client to the Realodex. 
 
-Format: `add n/NAME p/PHONE i/INCOME e/EMAIL a/ADDRESS f/FAMILY t/TAG ...`
+Format: `add n/NAME p/PHONE i/INCOME e/EMAIL a/ADDRESS f/FAMILY t/TAG [r/REMARK]`
```

Comment:
Did you put the [] to show that remark is optional?

Reply:
Yes

Reply:
Can highlight to show [] is optional thanks